charts/foundry/values.yaml

```diff
@@ -54,10 +54,6 @@ securityContext:
     drop:
     - ALL
   readOnlyRootFilesystem: false
-  runAsNonRoot: true
```

Comment:
@skudasov I had to remove this, otherwise Anvil is getting into permission denied error while creating dump state file

Reply:
AFAIR, if you remove this, your deployment would be blocked by a OPA gatekeeper policy engine. Were you able to deploy it?